shaderc-sys/build/build.rs

```diff
@@ -275,12 +276,17 @@ fn main() {
         let dylib_path = search_dir.join(dylib_name);
 
         if let Some((lib_name, lib_kind)) = {
-            if dylib_path.exists() {
-                Some((SHADERC_SHARED_LIB, "dylib"))
-            } else if static_lib_path.exists() {
-                Some((SHADERC_STATIC_LIB, "static"))
-            } else {
-                None
+            match (
+                dylib_path.exists(),
+                static_lib_path.exists(),
+                config_prefer_static_linking,
+            ) {
+                // if dylib not exist OR prefer static lib and static lib exist, static.
```

Comment:
Could you capitalize the first character? Similarly for the following comments (and then them with `.`).